app/forms/admin/schools/cohorts/appropriate_bodies/update_form.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module Admin
+  module Schools
+    module Cohorts
+      module AppropriateBodies
+        class UpdateForm
+          include ActiveModel::Model
+          include ActiveRecord::AttributeAssignment
+          include ActiveModel::Serialization
+
+          TEACHING_SCHOOL_HUB_ID = "teaching_school_hub"
+
+          attr_accessor :appropriate_body_id, :teaching_school_hub_id, :school_cohort
+
+          validates :appropriate_body_id, presence: { message: I18n.t("errors.appropriate_body.blank") }
+          validates :teaching_school_hub_id, presence: { message: I18n.t("errors.teaching_school_hub.blank") }, if: :teaching_school_hub_selected?
+          validate :validate_selected_appropriate_body
+
+          def save!
+            school_cohort.update!(appropriate_body_id: selected_appropriate_body_id)
```

Comment:
We have this SO to set it: Induction::SetSchoolCohortAppropriateBody

Reply:
I looked that one over and that'd update all of the existing induction records, the screenshots in the ticket indicated that this form shouldn't do that and should only apply for future participants.